.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
```

Comment:
I personally prefer monthly, rationale being that GHA version updates rarely provide much value but instead result in a lot of noisy PRs.